src/core/server/saved_objects/migrations/core/migrate_raw_docs.ts

```diff
@@ -71,6 +72,60 @@ export async function migrateRawDocs(
   return processedDocs;
 }
 
+/** TINA: approach options for https://github.com/elastic/kibana/issues/90279:
+ * don't log the errors
+ * return a list of processed docs and errors
+ * log those errors somewhere upstream in the next method that returns a complete list (using a new log message "corrupt-saved-object-${index-prefix}-${doc._id}" type)
+ */
+export interface DocumentsTransformFailed {
+  type: string;
+  failedDocumentIds: string[];
+}
+export interface DocumentsTransformSuccess {
+  processedDocs: SavedObjectsRawDoc[];
+}
+
+export function migrateRawDocsNonThrowing(
+  serializer: SavedObjectsSerializer,
+  migrateDoc: MigrateAndConvertFn,
+  rawDocs: SavedObjectsRawDoc[],
+  log: SavedObjectsMigrationLogger
+  // This method should never fail because we're returning an array of something, either transformed raw saved objects or an array of saved object ids.
+  // According to the docs we should be using a Task for process that will never fail
+): TaskEither.TaskEither<DocumentsTransformFailed, DocumentsTransformSuccess> {
+  return async () => {
```

Comment:
@rudolf I removed the try-catch block to try and figure out why `State` in the model is claiming "Function lacks ending return statement and return type does not include 'undefined'."
It didn't help 😢